backend/handler.js

```diff
@@ -66,3 +68,22 @@ module.exports.graphql = (event, context, cb) => {
         cb(null, {"errors": [err]});
       });
     }
+
+const xmlApp = express();
+
+xmlApp.get('/xml', (req, res) => {
+  // Setup connection to PostgresDB
+  const pgClient = new Client(process.env.PGCON);
+  pgClient.connect();
+
+  pgClient.query('select floods.legacy_xml()')
+    .then(pgres => 
+    {
+      res.type('.xml');
+      res.send(`<xml>${pgres.rows[0].legacy_xml}</xml>`);
```

Comment:
Is the `<xml>` prefix necessary? I'm concerned it may break existing legacy clients.

https://atxfloods.com/dashboard/phpsqlajax_genxml.php does not have the `<xml>` prefix.